src/transaction/log_prior_recv.cpp

```diff
@@ -104,6 +162,8 @@ namespace cublog
 	  }
 
 	ulock.lock ();
+
+	m_messages_consume_cv.notify_one ();
```

Comment:
This notification is still inside the protected critical section; swap with the `ulock.lock()` above?

Reply:
I missed. done.